### PR TITLE
Quarg Military Retaliation and General Improvement

### PR DESCRIPTION
**Pull request by Isaacssv552**
I've integrated my Quarg Military Retaliation plugin, which expands the Quarg's lineup of both outfits and ships in addition to adding a response to the player farming them. 

List of additions:
- The Quarg will hunt the player with increasingly powerful ships/fleets in response to the player's attacks. This will not happen until after the player has been warned and the Quarg have stationed defense fleets in their systems.
- The Quarg will station defense fleets in their systems as their initial response to the player's attacks. They will expand the fleets if the attacks continue.
- A Quarg Carrier and Quarg Fighter, both of which will spawn in response to the player's actions.
- Quarg medium and large warships, both of which will spawn in response to the player's actions. The medium warship will occasionally be seen patrolling, even in the absence of attacks.
- More descriptions for Quarg ships and outfits previously lacking them.
- Small and Large Graviton engines. Medium thrusters are tweaked to be more interested and less horrible.
- Antimatter core reenabled both for immersion and as a reward for overcoming the increased risk of Quarg farming.
- A special Quarg outfit that trades maneuverability for outfit space but is unplunderable and found only on their strongest ships, which also have self destruct ratings.
- A Quarg fixed fire weapon, focusing on ion and heat damage.
- A smaller, but less efficient, Quarg shield generator.
- Heavy versions of the Skylance and Anti-Missile.
- Added secondary functions to some Quarg outfits to explain their lack of certain basics such as ramscoops and fuel pods.
- Probably a few other things I forgot to mention.

### CODE REVIEW

**Comment by toilethinges** [comment]
All the angled quotations need to be replaced with standard quotations, to avoid style conflicts.

**Comment by Isaacssv552** [comment]
@toilethinges 
I'm using angle quotes to avoid messing up the quotes containing the text, is there a better way?
@Alkallid 
Thanks for the error-check.

File: data/quarg ships.txt [hunk 1]
```diff
@@ -96,4 +98,227 @@ ship "Quarg Wardragon"
...
+	turret -4 -10 "Quarg Anti-Missile" 
+	turret 8.5 -10.5 "Quarg Anti-Missile" 
+	engine -14 66
+	engine 19.5 65.5
+	explode "tiny explosion" 12
+	explode "small explosion" 16
+	explode "medium explosion" 30
+	explode "big explosion" 40
+	explode "huge explosion" 20
+	description "The Empyrealdragon is used by the Quarg as a medium warship. Although the Wardragon patrol craft is usually more than capable of dealing with any threat Empyrealdragons are still occasionally deployed."
```
Comment (Gkramme) [inline data/quarg ships.txt]: Comma after threat.

**Comment by Amazinite** [comment]
@Isaacssv552 use little hash marks instead of angled quotes. i'm on mobile right now, so I can't type them, but it's on the same key as the tilde (~). Or you could replace your double quotes within descriptions with single quotes.

**Comment by Alkallid** [comment]
Honestly, this won't make it in until you get actual individual sprites. Ask Wrzl or Mdstem for help.

**Comment by Isaacssv552** [comment]
@Alkallid 
Actual sprites would be great, I'll contact them.

**Comment by Wrzlprnft** [comment]
I can help, but since @Mdsmestad has done the elder dragon (as far as i know) it would better if he does the rest to keep one style.

**Comment by Isaacssv552** [comment]
@Wrzlprnft The Azure Ark and Battledrake are different enough in origin and mechanical style that a slightly different art style would actually be okay, as long as it's still based on the original Quarg style. Any outfit images you could provide would also be awesome. Thanks!

I'm also willing to integrate any sprites made not intended for existing ships and outfits.

File: data/quarg outfits.txt [hunk 2]
```diff
@@ -225,6 +225,7 @@ outfit "Medium Graviton Steering"
 	"turn" 1600
 	"turning energy" 10
 	"turning heat" 5
+	description "The Quarg's Medium Graviton Steering is strangely ineffective for such a technologically advanced engine. The Quarg's focus on reversible thrusters and beam turrets has led to a peculair situation. The turning speed of most vessels is considered mostly irrelevant."
```
Comment (toilethinges) [inline data/quarg outfits.txt]: Peculiar.

**Comment by jmhorjus** [comment]
I would love to have this as a plugin.

**Comment by Isaacssv552** [comment]
It is currently available as a plugin at https://groups.google.com/forum/#!category-topic/endless-sky/content-creation/A1U6UP0CKhE.

File: data/quarg missions.txt [hunk 3]
```diff
@@ -111,3 +111,288 @@ mission "First Contact: Kuwaru Efreti"
...
+				die
+			label converse
+			`"Why do you attack us? We can no longer tolerate your behavior, whatever the causes, and must destroy you. What is your decision?"`
+			choice
+				`"Alright, I will lift off in a moment." (Lie)`
+					goto flee
+				`"Alright, I will lift off in a moment."`
+					launch
+				`"No, I refuse. I don’t believe you’ll attack the city."`
+					’"That is regrettable."`
```
Comment (toilethinges) [inline data/quarg missions.txt]: Apostrophe instead of `.

**Comment by Isaacssv552** [comment]
@Mdsmestad, @Wrzlprnft, are either of you willing to help with the sprites/images?

**Comment by mdsmestad** [comment]
@Isaacssv552 There doesn't seem to be enough interest in this pluggin for now. I recommend taking it down, and perhaps we'll put some more work into it.

**Comment by Isaacssv552** [comment]
@mdsmestad 
Alright. I don't have much time to work on it right now, but I'll incorporate any new sprites you make.

**Pull request closed without merging**